### ISSUE TITLE
Chord bass with accidentals are ignored in the midi generation

### ISSUE THREAD

**Issue by TCPN** [paulrosen/abcjs#278]
When writing chords, we can specify the bass note after a slash.
However, if the bass note has an accidental, it will be ignored by the midi generation.
Here are some examples.
```
X: 1
M: 4/4
L: 1/4
K: E
"E" G4 |"B/D#"B4| "C#m/G#" E4 |-E4 |"A"c4|"E/G#"B4|"Bsus4"B4|-B4:|]
X: 2
M: 4/4
L: 1/4
K: Ab
"Ab"A4|"Bb/Ab"c2B2|"Db"A4|"Db/Eb"A2 G2:|]
```

**Comment by TCPN** [comment]
I believe this can be fixed by changing the codes near the end of the function `interpretChord` in the file `abc_midi_flattener.js`.

Maybe like this
```
var arr = remaining.split('/');
chick = chordNotes(bass, arr[0]);
if (arr.length === 2) {
	var explicitBass = basses[arr[1].substring(0,1)];
	if (explicitBass) {
		var bassAcc = arr[1].substring(1);
		var bassShift = {'#': 1, '♯': 1, 'b': -1, '♭': -1}[bassAcc] || 0;
		bass = basses[arr[1].substring(0,1)] + bassShift + transpose;
		bass2 = bass;
	}
}
```

**Comment by paulrosen** [comment]
Thanks for the bug report. Do you want to do a pull request or would you like me to just fix it?

**Comment by TCPN** [comment]
So..I did a pull request. Please review and merge it if you think it's ok!
Thanks!

**Comment by paulrosen** [comment]
That looks good - I hope to have a minor release fairly soon and I'll include it in that. When testing that I found another couple of small bugs so thanks for the inspiration!

**Comment by paulrosen** [comment]
Fixed in version 5.11.0
